cloud_computing/cloud_computing_intro/index.md

```diff
@@ -0,0 +1,76 @@
+---
+name: Introduction to Cloud Computing
```

Comment:
I would generally shorten this introduction section. Benefits/drawbacks of cloud are well known at this point, and having a couple of paragraphs should be enough.